_includes/tutorials/aggregating-average/kstreams/code/build.gradle

```diff
@@ -0,0 +1,75 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.15.1"
+    classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+  }
+}
+
+plugins {
+  id "java"
+  id "application"
+  id "com.google.cloud.tools.jib" version "1.1.1"
+  id "idea"
+  id "eclipse"
+}
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+version = "0.0.1"
+mainClassName = "io.confluent.developer.RunningAverage"
+
+repositories {
+  jcenter()
+  maven { url 'https://packages.confluent.io/maven/' }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+  implementation "org.apache.avro:avro:1.8.2"
+  implementation "org.slf4j:slf4j-simple:1.7.26"
+  implementation "org.apache.kafka:kafka-streams:2.4.0"
```

Comment:
can we update to `2.4.1`? Same for the `streams-test-utils`

Reply:
done